test/libsolidity/syntaxTests/largeTypes/oversized_contract.sol

```diff
@@ -6,5 +6,3 @@ contract C {
 }
 // ----
 // TypeError 7676: (60-114): Contract too large for storage.
-// Warning 3408: (77-91): Variable "a" covers a large part of storage and thus makes collisions likely. Either use mappings or dynamic arrays and allow their size to be increased only in small quantities per transaction.
```

Comment:
These are now removed because the other checks are done at a later stage and the compiler aborts already.